### PR TITLE
Following count of a user #10

### PR DESCRIPTION
**Pull request by codesleep-Beperfect**
following method added in class Users

### CODE REVIEW

File: src/scrape_up/github/users.py [hunk 1]
```diff
@@ -184,53 +196,3 @@ def get_starred_repos(self):
...
-            for user in followers_body.find_all('span', class_='Link--secondary'):
-                followers.append(user.text.strip())
-            
-            return followers
-        except:
-            message = f"Followers not found for username {self.username}"
-            return message
-    
-    def __scrape_following_page(self):
-        """
```
Comment (nikhil25803) [inline src/scrape_up/github/users.py:214]: Why you deleted all these methods? @codesleep-Beperfect

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/github/users.py (hunk 1 above)
Comment (codesleep-Beperfect) [reply src/scrape_up/github/users.py:214]: it was deleted while resolving merges
@nikhil25803 should i recreate the pull request?

Comment (nikhil25803) [reply src/scrape_up/github/users.py:214]: That would be better, create a new PR.

Comment (codesleep-Beperfect) [reply src/scrape_up/github/users.py:214]: Okay Cool

**Pull request closed without merging**